app/assets/stylesheets/argo/impersonate_groups.scss

```diff
@@ -0,0 +1,12 @@
+.auth-groups {
+  margin: 0 auto;
+  width: 500px;
+}
+
+.navbar-nav > li > a.impersonating {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3